t/ui/18-tests-details.t

```diff
@@ -57,7 +58,8 @@ t::ui::PhantomTest::wait_for_ajax;
 ok($driver->find_element('#preview_container_out', 'css')->is_displayed(), "preview window opens on click");
 like($driver->find_element('#preview_container_in', 'css')->get_text(), qr/wait_serial expected/, "Preview text with wait_serial output shown");
 $driver->find_element('[title="wait_serial"]', 'css')->click();
-t::ui::PhantomTest::wait_for_ajax;
+# wait for animation to finish
+sleep .2;
```

Comment:
ok , good. The problem I see is that reading the code there is no hint that this number is exactly the time of the animation. Your explanation makes perfect sense but reading the code I still see the problem of magic numbers. I can think of some ideas:
* Read the animation timeout from a variable to be used within the production code as well as the test (can be very clean)
* Set animation timeout to 0 for testing purposes (saves time when executing the tests)
* create a variable with expressive name resembling your explanation, e.g. `my animation_duration = 0.2; # see path/to/wherever/animation/used.js` (easy but [comments lie](http://blog.ircmaxell.com/2012/06/to-comment-or-not-to-comment-tale-of.html))